README.md

```diff
@@ -56,23 +59,23 @@ To match something with spaces, it must be wrapped in parenthesis:
 (a phrase has spaces)
 ```
 
-This goes for the `[server name]` and `#[channel name]` as well, as those both are treated as patterns, too.
+This applies the same for `[server name]` and `#[channel name]` as well.
 
-Note that the `#` for a `[channel name]` will be on the outside of the parenthesis:
+Note that the `#` for a `[channel name]` *must* be on the outside of the parenthesis:
 
 ```
 #(channel name)
 ```
 
-This is in part because the `[server name]`, `[channel name]`, and `[pattern]` are all (usually) treated as [Lua patterns][lua-patterns].
+This is because the `[server name]`, `[channel name]`, and `[pattern]` are all (usually) treated as [Lua patterns][lua-patterns].
 
-This means that, a `[pattern]` of `word` will match any message with `word` in it, like `Oh my word, that's awesome!`.
+As an example, a `[pattern]` of `word` will match any message with `word` in it, like `Oh my word, that's awesome!`.
 
 This also has a few consequences:
 
-Lua patterns treat `%` `(` `)` `.` `[` `]` `*` `+` `-` `?` all as special characters that do different things. Additionally, `^` is treated as special when it's at the beginning of a `[pattern`], and `$` when it's at the end.
+Lua patterns treat `%` `(` `)` `.` `[` `]` `*` `+` `-` `?` all as special characters that do different things. Additionally, `^` is treated as special when it's at the beginning of a `[pattern]`, and `$` when it's at the end.
 
-To have Lua treat them as regular characters, put a `%` before them.
+To have Lua treat any of these as regular characters, put a `%` before them.
 
 As an example, to match the text `:-) tehee (-:`, the whole thing first needs to be wrapped in parenthesis, and the `(`, `)`, and `-` inside need to be escaped with a `%`:

```

Comment:
```
As an example, to match the text `:-) tehee (-:`, the `[pattern]` first needs to be wrapped in parentheses, and the `(`, `)`, and `-` that are part of the `[pattern]` need to be escaped with a `%`:
```

Reply:
References to `[pattern]` here aren't useful.